Inventory.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace InventoryManagement
+{
+    public class Inventory
+    {
+
+        List<Product> products = new List<Product>();
+
+        //Define Constructor 
```

Comment:
are these comments really useful? doesn't the method name and its signature convey this information?